### PR TITLE
Type annotations to Koalas accessors and Spark accessors

### PR DESCRIPTION
**Pull request by xinrong-databricks**

### CODE REVIEW

File: databricks/koalas/accessors.py [hunk 1]
```diff
@@ -200,7 +200,7 @@ def apply_batch(self, func, args=(), **kwds):
 
             >>> # This case does not return the length of whole frame but of the batch internally
             ... # used.
-            ... def length(pdf) -> ks.DataFrame[int]:
+            ... def length(pdf) -> pd.DataFrame[int]:
```
Comment (ueshin) [inline databricks/koalas/accessors.py:203]: Can we change this?

File: databricks/koalas/accessors.py [hunk 2]
```diff
@@ -389,7 +389,7 @@ def apply_batch(self, func, args=(), **kwds):
 
         return DataFrame(internal)
 
-    def transform_batch(self, func, *args, **kwargs):
+    def transform_batch(self, func, *args, **kwargs) -> "DataFrame":
```
Comment (ueshin) [inline databricks/koalas/accessors.py:392]: `transform_batch` can return `Series` as well.

Comment (xinrong-databricks) [reply databricks/koalas/accessors.py]: Makes sense! Updated.

**Review by ueshin — APPROVED** [review]
LGTM, pending tests.

**Comment by xinrong-databricks** [comment]
@ueshin Thank you!

**Pull request merged**